src/index.js

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { BrowserRouter } from 'react-router-dom';
+// component file
+import TodoContainer from './functionBased/components/TodoContainer';
```

Comment:
Here in the index.js file you are using the functionalBased component, I Kindly suggest to remove the classbased component because there is clear use for it in the project.
![image](https://user-images.githubusercontent.com/114300121/213262764-7f2c1ab3-940c-468e-bea1-ea705336fd17.png)